#ifndef ADDSTUDENTWIDGET_H
#define ADDSTUDENTWIDGET_H

#include <QWidget>
#include<QMessageBox>//对话框
#include<QFile>
#include<QDebug>

namespace Ui {
class AddStudentWidget;
}

class AddStudentWidget : public QWidget
{
    Q_OBJECT

public:
    explicit AddStudentWidget(QWidget *parent = nullptr);
    ~AddStudentWidget();

private:
    Ui::AddStudentWidget *ui;

signals:
    void display(int number);

private slots:
    void on_returnButton_clicked();
    void on_addButton_clicked();
};

#endif // ADDSTUDENTWIDGET_H
